akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingEvents.scala

```diff
@@ -104,6 +107,7 @@ private[persistence] class ReplayingEvents[C, E, S](override val setup: Behavior
           Behaviors.unhandled
       }
     } catch {
+      case FailureWhileUnstashing(ex) ⇒ throw ex
```

Comment:
that looks right, throwing in same way as if the command would be received later in the Running